### PR TITLE
[Retouch] unittest_models test cases

### PR DESCRIPTION
**Pull request by zhoonit**
- [Retouch] unittest_models test cases

```
**Changes proposed in this PR:**
- Reduce scale of inputs
- Add iterations for recurrent layers (and delete _n) family tcs
- Clean duplicated codes in genModelTests.py

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Comment by zhoonit** [comment]
Code merged with https://github.com/nnstreamer/nntrainer/pull/1220

File: test/unittest/unittest_nntrainer_models.cpp [hunk 1]
```diff
@@ -1235,19 +1191,14 @@ INSTANTIATE_TEST_CASE_P(
...
     // mkModelTc(fc_softmax_cross_distribute_validate, "3:1:5:3", 1),
     // mkModelTc(fc_sigmoid_cross_distribute_validate, "3:1:5:3", 1)
-    mkModelTc(rnn_basic, "1:1:1:1", 1),
-    mkModelTc(rnn_return_sequences, "1:1:2:1", 1),
-    mkModelTc(lstm_basic, "1:1:1:1", 1),
-    mkModelTc(lstm_return_sequence, "1:1:2:1", 1),
-    mkModelTc(lstm_return_sequence_with_batch, "2:1:2:1", 1),
-    mkModelTc(lstm_return_sequence_with_batch_n, "2:1:2:1", 2),
-    mkModelTc(multi_lstm_return_sequence, "1:1:1:1", 1),
-    mkModelTc(multi_lstm_return_sequence_with_batch_n, "2:1:1:1", 2),
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:1245]: I wonder why remove negative testcases from here?

File: test/unittest/unittest_nntrainer_models.cpp [hunk 2]
```diff
@@ -1235,19 +1191,14 @@ INSTANTIATE_TEST_CASE_P(
...
-    mkModelTc(multi_rnn_return_sequence_with_batch_n, "2:1:1:1", 2)
-
+    mkModelTc(lstm_basic, "1:1:1:1", 10),
+    mkModelTc(lstm_return_sequence, "1:1:2:1", 10),
+    mkModelTc(lstm_return_sequence_with_batch, "2:1:2:1", 10),
+    mkModelTc(multi_lstm_return_sequence, "1:1:1:1", 10),
+    mkModelTc(rnn_basic, "1:1:1:1", 10),
+    mkModelTc(rnn_return_sequences, "1:1:2:1", 10),
+    mkModelTc(rnn_return_sequence_with_batch, "2:1:2:1", 10),
+    mkModelTc(multi_rnn_return_sequence, "1:1:1:1", 10)
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:1201]: mutli_rnn_return_(sequence_)_with_batch is missing.

Comment (zhoonit) [reply test/unittest/unittest_nntrainer_models.cpp:1201]: Maybe there was a reason, but I don't see why I deleted. I'll make it revive in a new PR.

File: test/unittest/unittest_nntrainer_models.cpp (hunk 1 above)
Comment (zhoonit) [reply test/unittest/unittest_nntrainer_models.cpp:1245]: here _n is not negative case but denotes n-iteration.

n-iteration is not needed as the patch makes all the test have 10 iterations.

**Pull request closed without merging**